LogGenerator/src/main/java/com/loggenerator/LogGenerator.java

```diff
@@ -0,0 +1,166 @@
+        "O usuario {CPF} finalizou sua compra em {DATETIME}.",
+        "O usuario {CPF} enviou uma mensagem para o usuario {TO_USER} em {DATETIME}.",
+        "O usuario {CPF} compartilhou {CONTENT} com seus seguidores em {DATETIME}.",
+        "O usuario {CPF} fez o download de {FILE_NAME} em {DATETIME}.",
+        "Um novo produto foi adicionado ao inventario em {DATETIME}.",
+        "Um pedido foi feito pelo cliente {CUSTOMER_ID} em {DATETIME}.",
+        "Um pagamento de {AMOUNT} foi processado em {DATETIME}.",
+        "Um novo post de blog foi publicado em {DATETIME}.",
+        "Um novo evento foi agendado para {DATE} as {TIME}.",
+        "Uma nova candidatura de emprego foi recebida em {DATETIME}.",
+        "Uma nova mensagem foi recebida de {SENDER_NAME} em {DATETIME}.",
+        "Uma nova assinatura foi criada por {CUSTOMER_NAME} em {DATETIME}.",
+        "Um novo arquivo foi enviado para o servidor em {DATETIME}.",
+        "Uma nova venda foi criada em {DATETIME}.",
+        "Um novo plano de assinatura foi criado em {DATETIME}.",
+        "Uma nova vaga de emprego foi publicada em {DATETIME}.",
+        "Um novo relatorio de erro foi enviado em {DATETIME}.",
+        "Uma nova avaliacao de produto foi enviada por {USER_NAME} em {DATETIME}.",
+        "Uma nova inscricao em evento foi recebida de {USER_NAME} em {DATETIME}.",
+        "Uma nova consulta foi agendada para {DATE} as {TIME}."};
+
+    private static String generateDate(Random random) {
+        Calendar calendar = Calendar.getInstance();
+
+        // Gerando ano aleatorio entre 1950 e 2023
+        int year = random.nextInt(2) + 2023;
+        // Gerando mês aleatorio (valores vao de 0 a 11)
+        int month = random.nextInt(12);
+        // Gerando dia aleatorio (valores vao de 1 a 28, 29, 30 ou 31)
+        int day = random.nextInt(calendar.getActualMaximum(Calendar.DAY_OF_MONTH)) + 1;
+
+        // Configurando data gerada no calendario
+        calendar.set(year, month, day);
+
+        // Obtendo data no formato desejado
+        SimpleDateFormat dateFormat = new SimpleDateFormat("dd/MM/yyyy");
+        String data = dateFormat.format(calendar.getTime());
+        return data;
+    }
+
+    public static String generateHours(Random random) {
+        int hora = random.nextInt(24);
+        int minuto = random.nextInt(60);
+
+        String horaFormatada = String.format("%02d:%02d", hora, minuto);
+        return horaFormatada;
+    }
+
+    private static String generateCPF(Random random) {
+        int n1 = random.nextInt(10);
+        int n2 = random.nextInt(10);
+        int n3 = random.nextInt(10);
+        int n4 = random.nextInt(10);
+        int n5 = random.nextInt(10);
+        int n6 = random.nextInt(10);
+        int n7 = random.nextInt(10);
+        int n8 = random.nextInt(10);
+        int n9 = random.nextInt(10);
+        int d1 = 11 - ((n1 * 10 + n2 * 9 + n3 * 8 + n4 * 7 + n5 * 6 + n6 * 5 + n7 * 4 + n8 * 3 + n9 * 2) % 11);
```

Comment:
We can create a function that performs this computation. Please use a loop instead of hardcoding this many variables. This looks super verbose, to be honest